OpenEdXMobile/res/layout-land/layout_error_state.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we can use the single `layout_error_state` and not need to create for the `landscape` mode.

Reply:
removed